app/client/views/activities/pagination/pagination.css

```diff
@@ -0,0 +1,12 @@
+.width {
+  width: 80px !important;
```

Comment:
If possible, we should avoid hard-coding layout in pixel values. Considering using a relative value such as `em` or `rem`.